lib/bamboo/test.ex

```diff
@@ -82,31 +82,28 @@ defmodule Bamboo.Test do
   checked when using the assertion helpers like `assert_delivered_email/1`.
 
   Sometimes emails don't show up when asserting because you may deliver an email
-  from a _different_ process than the test process. When that happens, set the
-  `process_name`. This will name the test process using `Process.register/2`
-  and set `Bamboo.TestAdapter` to always send to that process. This means
-  that you cannot use `process_name` with async tests.
+  from a _different_ process than the test process. When that happens, turn on
+  shared mode. This will tell `Bamboo.TestAdapter` to always send to the test process.
+  This means that you cannot use shared mode with async tests.
 
   ## Try to use this version first
 
       use Bamboo.Test
 
-  ## And if you are delivering from another process, set `process_name`
-
-      # Note: the process name can be whatever you want.
-      use Bamboo.Test, process_name: :name_of_my_test
+  ## And if you are delivering from another process, set `shared: true`
 
+      use Bamboo.Test, shared: true
 
   Common scenarios for delivering mail from a different process are when you
   send from inside of a Task, GenServer, or are running acceptance tests with a
   headless browser like phantomjs.
   """
-  defmacro __using__(process_name: process_name) do
+  defmacro __using__(shared: true) do
     quote do
       setup tags do
         if tags[:async] do
           raise """
-          passing process_name to Bamboo.Test cannot be done for async tests.
+          using shared mode with Bamboo.Test cannot be done for async tests.
```

Comment:
Oh never mind that should not be capitalized. The beginning of errors are supposed to be lowercased because the sentence starts with `raised RuntimeError` or something like that. Here's an example: https://github.com/phoenixframework/phoenix/blob/e3c23a364af4f2283e2498fdb47c60161f262a1f/lib/phoenix/router.ex#L432